src/it/scala/io/iohk/ethereum/sync/RegularSyncItSpec.scala

```diff
@@ -12,20 +16,44 @@ import scala.concurrent.duration._
 class RegularSyncItSpec extends FreeSpecBase with Matchers with BeforeAndAfterAll {
   implicit val testScheduler = Scheduler.fixedPool("test", 16)
 
+  override def beforeAll(): Unit = {
+    Metrics.configure(
+      MetricsConfig(Config.config.withValue("metrics.enabled", ConfigValueFactory.fromAnyRef(true)))
+    )
+  }
+
   override def afterAll(): Unit = {
     testScheduler.shutdown()
     testScheduler.awaitTermination(120.second)
   }
 
+  val getMinedBlockValue = CollectorRegistry.defaultRegistry.getSampleValue(
+    _: String,
+    Array("class"),
+    Array("MinedBlockPropagation")
+  )
+
+  val getNewBlockValue = CollectorRegistry.defaultRegistry.getSampleValue(
+    _: String,
+    Array("class"),
+    Array("NewBlockPropagation")
+  )
+
+  val getDefaultBlockValue = CollectorRegistry.defaultRegistry.getSampleValue(
+    _: String,
+    Array("class"),
+    Array("DefaultBlockPropagation")
+  )
```

Comment:
maybe we could start adding helpers that make it easier to read? eg.
```scala
MantisRegistries.sampleClass("DefaultBlockPropagation")

object MantisRegistries {
  def sampleClass(className: String) = CollectorRegistry.defaultRegistry.getSampleValue(
    _: String,
    Array("class"),
    Array(className)
  )
}
```

Reply:
I have added the helper, the code is more clean indeed. I renamed the label name from `class` to `blocktype`